src/agent.cpp

```diff
@@ -162,6 +165,14 @@ int Agent::getUnum() {
   return world().self().unum();
 }
 
+void Agent::setLastActionStatusCollision(bool may_fix, bool likely_success) {
```

Comment:
Note also that I have been concentrating on the actions available to the offense; collisions with the ball by the defense tend to immediately end the game, for instance.